src/_includes/social-proof.njk

```diff
@@ -3,6 +3,7 @@
     <li class="w-[135px] sm:w-[156px]">{% image "./images/home-logos/leclanche-logo.png", "Leclanché logo", [156] %}</li>
     <li class="w-[70px] sm:w-[93px]">{% image "./images/home-logos/lilly-logo.svg", "Lilly logo", [113] %}</li>
     <li class="w-[113px] sm:w-[131px]">{% image "./images/home-logos/bosch-logo.png", "Bosch logo", [131] %}</li>
+    <li class="w-[46px] sm:w-[62px] mix-blend-luminosity saturate-0 opacity-80">{% image "./images/home-logos/sky-logo.svg", "Sky logo", [131] %}</li>
```

Comment:
Do we want to drop a logo to make some space?

Reply:
Sky's logo is narrow, I don't think we need to remove one